pkg/apis/configuration/validation/virtualserver.go

```diff
@@ -103,7 +105,7 @@ func validatePolicies(policies []v1.PolicyReference, fieldPath *field.Path, name
 	return allErrs
 }
 
-func validateTLS(tls *v1.TLS, fieldPath *field.Path) field.ErrorList {
+func validateTLS(isCertManagerEnabled bool, tls *v1.TLS, fieldPath *field.Path) field.ErrorList {
```

Comment:
If these were methods then they'd have access to the Validator state and not require SDK signature changes.